org.civicrm.hremergency/CRM/Hremergency/Upgrader/Steps/1000.php

```diff
@@ -14,7 +14,7 @@ public function upgrade_1000() {
 
     // create required options
     foreach($relationships as $relationship) {
-      if (!$this->up1000_valueExists($relationship, 'relationship_with_employee_20150304120408')) {
+      if (!$this->up1000_isExistingOptionValue($relationship, 'relationship_with_employee_20150304120408')) {
```

Comment:
I think up1000_isOptionValueExist make more sense and even shorter than up1000_isExistingOptionValue